exampleSite/config/_default/params.toml

```diff
@@ -182,6 +182,9 @@ cacheRemoteImages = false
   # 是否启用阅读时间估计
   enableReadingTime = true
   # Display a message at the beginning of an article to warn the reader that its content might be outdated.
```

Comment:
This comment should stay together with the comment in line 188 as it is the English translation of that Chinese description